channeldb/db.go

```diff
@@ -142,7 +142,9 @@ type DB struct {
 
 // Open opens an existing channeldb. Any necessary schemas migrations due to
 // updates will take place as necessary.
-func Open(dbPath string, modifiers ...OptionModifier) (*DB, error) {
+func Open(dbPath string, now func() time.Time, modifiers ...OptionModifier) (
```

Comment:
why not set `time.Now` as the default in `DefaultOptions()`, then add a functional modifier? much smaller diff

Reply:
Great suggestion, done